src/lib/state/settings.ts

```diff
@@ -1,34 +1,26 @@
+import { Percent } from '@uniswap/sdk-core'
 import { atomWithReset } from 'jotai/utils'
 
-import { Customizable, pickAtom, setCustomizable, setTogglable } from './atoms'
```

Comment:
I believe you can delete all of the `Customizable` code at this point.

Reply:
If there's something using it that is not in the UI (tech debt) you can put this off to another PR / I can get rid of it after this merges.